src/rust/src/padding.rs

```diff
@@ -110,6 +110,62 @@ impl PKCS7PaddingContext {
     }
 }
 
+#[pyo3::pyclass]
```

Comment:
For this migration, this can go into its own PR -- it can be merged without any of the rest of the freethreading stuff.

Reply:
I'll cherrypick the Rust ports and make them their own PRs and keep the free-threading-specific stuff in this PR.